play-services-fido/core/src/main/kotlin/org/microg/gms/fido/core/transport/nfc/NfcTransportHandler.kt

```diff
@@ -6,15 +6,16 @@
 package org.microg.gms.fido.core.transport.nfc
 
 import android.app.Activity
+import android.app.ActivityOptions
 import android.app.PendingIntent
 import android.content.Intent
 import android.content.IntentFilter
 import android.nfc.NfcAdapter
 import android.nfc.Tag
 import android.nfc.tech.IsoDep
+import android.os.Build
```

Comment:
Hi, it was decided that:
- For fake data (org.microg.gms.profile.Build) import all
- For real data (android.os.Build)  import just the specific fields directly (like android.os.Build.VERSION.SDK_INT)

Reply:
OK, I've updated the patch